internal/controller/postgrescluster/pgbackrest.go

```diff
@@ -299,10 +298,8 @@ func (r *Reconciler) cleanupRepoResources(ctx context.Context,
 				owned.GetName() != naming.PGBackRestSSHSecret(postgresCluster).Name {
```

Comment:
There's a TODO on line 296/7 that might be worth handling.

Reply:
I think that's true, but it's a little beyond the scope of this story. I think it's good to get this PR merged now, so I'll leave that in for the time being.